samples/tab-sso/nodejs/src/client/scripts/ssoDemo.js

```diff
@@ -88,41 +89,47 @@
     }
 
     // In-line code
-    getClientSideToken()
-        .then((clientSideToken) => {
-            return getServerSideToken(clientSideToken);
-        })
-        .then((profile) => {
-            return useServerSideToken(profile);
-        })
-        .catch((error) => {
-            if (error === "invalid_grant") {
-                display(`Error: ${error} - user or admin consent required`);
-                // Display in-line button so user can consent
-                let button = display("Consent", "button");
-                button.onclick = (() => {
-                    requestConsent()
-                        .then((result) => {
-                             // Consent succeeded
-                             display(`Consent succeeded`);
-                            
-                             // offer to refresh the page
-                             button.disabled = true;
-                             let refreshButton = display("Refresh page", "button");
-                             refreshButton.onclick = (() => { window.location.reload(); });
-                        })
-                        .catch((error) => {
-                            display(`ERROR ${error}`);
-                            // Consent failed - offer to refresh the page
-                            button.disabled = true;
-                            let refreshButton = display("Refresh page", "button");
-                            refreshButton.onclick = (() => { window.location.reload(); });
+    $(document).ready(function () {
+        microsoftTeams.app.initialize().then(() => {
+            getClientSideToken()
+                .then((clientSideToken) => {
+                    return getServerSideToken(clientSideToken);
+                })
+                .then((profile) => {
+                    return useServerSideToken(profile);
+                })
+                .catch((error) => {
+                    if (error === "invalid_grant") {
+                        display(`Error: ${error} - user or admin consent required`);
+                        // Display in-line button so user can consent
+                        let button = display("Consent", "button");
+                        button.onclick = (() => {
+                            requestConsent()
```

Comment:
Add proper spacing and line break before if, let etc.

Reply:
Added